my_pygame.py

```diff
@@ -89,8 +89,7 @@ def draw_buttons():
             for button in list_of_ingredients:
                 if x >= button.xcoor and x <= button.xcoor + button.width and y >= button.ycoor and y <= button.ycoor + button.height:
```

Comment:
You can reduce this line by using combined comparison operators.
button.xcoor >= x <= button.xcoor.